README.md

```diff
@@ -31,35 +41,43 @@ npm install --save ts-json-schema-generator
+By default, the command-line generator will use the `tsconfig.json` file in the
+current working directory, or the first parent directory that contains a
+`tsconfig.json` file up to the root of the filesystem. If you want to use a
+different `tsconfig.json` file, you can use the `--tsconfig` option. In
+particular, if you need to use different compilation options for types, you may
+want to create a separate `tsconfig.json` file for the schema generation only.
 
 ### Options
 
 ```
-  -p, --path <path>              Source file path
-  -t, --type <name>              Type name
-  -i, --id <name>                $id for generated schema
-  -f, --tsconfig <path>          Custom tsconfig.json path
-  -e, --expose <expose>          Type exposing (choices: "all", "none", "export", default: "export")
-  -j, --jsDoc <extended>         Read JsDoc annotations (choices: "none", "basic", "extended", default: "extended")
-  --markdown-description         Generate `markdownDescription` in addition to `description`.
-  --functions <functions>        How to handle functions. `fail` will throw an error. `comment` will add a comment. `hide` will treat the function like a NeverType or HiddenType.
-                                 (choices: "fail", "comment", "hide", default: "comment")
-  --minify                       Minify generated schema (default: false)
-  --unstable                     Do not sort properties
-  --strict-tuples                Do not allow additional items on tuples
-  --no-top-ref                   Do not create a top-level $ref definition
-  --no-type-check                Skip type checks to improve performance
-  --no-ref-encode                Do not encode references
-  -o, --out <file>               Set the output file (default: stdout)
-  --validation-keywords [value]  Provide additional validation keywords to include (default: [])
-  --additional-properties        Allow additional properties for objects with no index signature (default: false)
-  -V, --version                  output the version number
-  -h, --help                     display help for command
+-p, --path <path>              Source file path
+-t, --type <name>              Type name
+-i, --id <name>                $id for generated schema
+-f, --tsconfig <path>          Custom tsconfig.json path
+-e, --expose <expose>          Type exposing (choices: "all", "none", "export", default: "export")
+-j, --jsDoc <extended>         Read JsDoc annotations (choices: "none", "basic", "extended", default: "extended")
+--markdown-description         Generate `markdownDescription` in addition to `description`.
+--functions <functions>        How to handle functions. `fail` will throw an error. `comment` will add a comment. `hide` will treat the function like a NeverType or HiddenType.
+                               (choices: "fail", "comment", "hide", default: "comment")
+--definition-titles            Generates titles from definition names
```

Comment:
I'm pretty sure your formatter changed this whole file, could you just add this line and remove the remaining formatting changes?